docs/startleft-processors/external-threat-model/mtmt/MTMT-Mapping.md

```diff
@@ -113,7 +113,142 @@ the mapping file. Let's see how to do it:
 ## Dataflows
 No need to be in the mapping file. StartLeft will detect them and will fill the OTM dataflow automatically detecting
 the custom name, the source, and the destination
-## Threats
-No need to be in the mapping file. StartLeft will detect them and will fill the OTM threats automatically
-## Mitigations
-No need to be in the mapping file. StartLeft will detect them and will fill the OTM mitigations automatically
\ No newline at end of file
+
+## Threats and Mitigations
+No need to be in the mapping file.
+
+StartLeft will extract OTM threats and threat instances, as well as OTM mitigations
+and mitigation instances from the same MTMT threat. Moreover the OTM component to which the OTM threat instance will be added
+is the destination component from the MTMT threat.
```

Comment:
"is the destination component **`of`** the MTMT threat."

Reply:
Done!